spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/spark/streaming/MessageBusConfiguration.java

```diff
@@ -54,6 +56,10 @@
 	 */
 	static ConfigurableApplicationContext createApplicationContext(final Properties properties) {
 		String transport = properties.getProperty("XD_TRANSPORT");
+		if (properties.getProperty(RABBIT_ACKMODE_PROPERTY) != null) {
+			properties.setProperty(RABBIT_ACKMODE_PROPERTY, "MANUAL");
+		}
```

Comment:
We copy the message bus properties from XD configuration (along with what end user provides) and set here for Spark. At this level we don't know if the property comes from XD config or user level settings. That's the reason, we always override irrespective what's being set already. What do you think?